meta-mender-core/recipes-bsp/u-boot/u-boot-mender-common.inc

```diff
@@ -368,6 +372,24 @@ python() {
 # Helpers and internal variables.
 ################################################################################
 
+# @return the device portion of the uboot environment path (IE mmcblk0boot1, mmcblk0boot0 or mmcblk0)
+def mender_get_uboot_env_mmc_linux_device(d):
+    # Convert u-boot partition numbers to linux device references
+    # See [CONFIG_SYS_MMC_ENV_PART](https://github.com/u-boot/u-boot/blob/u-boot-2016.09.y/README#L4390) and
+    # [linux kernel mmc-dev-parts documentation](https://www.kernel.org/doc/Documentation/mmc/mmc-dev-parts.txt)
+    storage_device=d.getVar("MENDER_STORAGE_DEVICE",True)
+    env_part=d.getVar("MENDER_UBOOT_CONFIG_SYS_MMC_ENV_PART",True)
+    env_part_append=""
+    if env_part == "1":
+        env_part_append="boot0"
+    elif env_part == "2":
+        env_part_append="boot1"
+    return storage_device.replace("/dev/","",1) + env_part_append
+
+# Does not include the /dev prefix
+MENDER_UBOOT_MMC_ENV_LINUX_DEVICE ?= "${@mender_get_uboot_env_mmc_linux_device(d)}"
+MENDER_UBOOT_MMC_ENV_LINUX_DEVICE_PATH ?= "/dev/${MENDER_UBOOT_MMC_ENV_LINUX_DEVICE}"
```

Comment:
`MENDER_UBOOT_MMC_ENV_LINUX_DEVICE` is not used anywhere. I propose removing it, and using only `MENDER_UBOOT_MMC_ENV_LINUX_DEVICE_PATH`.

Reply:
MENDER_UBOOT_MMC_ENV_LINUX_DEVICE is used in the meta-tegra layer, when setting up fw_env.conf.  I've [moved the definition there](https://github.com/Trellis-Logic/meta-mender-community/pull/1/files#diff-60715b5b8ad1f73149b080bbdac6f4a1R12).

Addressed in [force push](https://github.com/mendersoftware/meta-mender/compare/f92f8114ca70418350489e978f3ae198f578d045..f65ae24861b63bcd85e22fb9fa08236339d9a801), also updated commit comment.